### ISSUE TITLE
Spotlight page indicator doesn't change page

### ISSUE THREAD

**Issue by simonmitchell** [3sidedcube/ThunderCloud#196]
## Expected Behaviour

The spotlight page indicator should be able to change the visible page of the spotlight, as otherwise users using VoiceOver can't change the selected page of the spotlight using swipe up and down.

## Current Behaviour

Swiping up or down whilst in VoiceOver on the spotlight page indicator changes the current page of the page indicator, but doesn't scroll the spotlight.

## Possible Solution

Add target and selector to page indicator to scroll to the selected page.

## Steps to Reproduce (for bugs)

1. Enable VoiceOver on iOS
2. Visit page with spotlight
3. Tap page indicator or spotlight itself
4. Observe can't change visible page
